components/calendar/src/any_calendar.rs

```diff
@@ -39,14 +39,14 @@ use core::fmt;
 /// There are many ways of constructing an AnyCalendar'd date:
 /// ```
 /// use icu::calendar::{AnyCalendar, AnyCalendarKind, DateTime, japanese::Japanese, types::Time};
-/// use icu::locid::Locale;
+/// use icu::locid::{Locale,locale};
```

Comment:
Nit, here and elsewhere: I think you don't need to import `Locale` any more. Just `locale`. (I won't comment on every spot)

Reply:
Done.